src/main/ml-modules/root/lib/environmentLib.mjs

```diff
@@ -225,7 +225,12 @@ function getStorageInfo() {
 function _getDataConversionDate() {
   try {
     return fn

```

Comment:
@clarkepeterf, the following was changed as we can no longer assume the requesting user has access to the first document identified by cts.uris.